src/Network/Skylark/Core/Types.hs

```diff
@@ -217,85 +214,86 @@ instance Var LogLevel where
 
 instance (Var a, Show a, Read a) => Var (Maybe a) where
   toVar (Just v) = (unpack . show) v
-  toVar Nothing = ""
+  toVar Nothing  = ""
+
   fromVar = readMaybe
 
 -- | A record type representing full or partial configuration of an
 -- HTTP service. Remaining unspecified fields are filled in with the
 -- default values from Default.
 --
-data Config = Config
-  { _cConfigFile    :: Maybe String   -- ^ Service configuration file location
-  , _cPort          :: Maybe Word32   -- ^ Port to listen on
-  , _cTimeout       :: Maybe Word32   -- ^ Connection timeout (sec)
-  , _cLogLevel      :: Maybe LogLevel -- ^ Logging level
+data Conf = Conf
+  { _confFile     :: Maybe String   -- ^ Service configuration file location
+  , _confPort     :: Maybe Int      -- ^ Port to listen on
+  , _confTimeout  :: Maybe Int      -- ^ Connection timeout (sec)
+  , _confLogLevel :: Maybe LogLevel -- ^ Logging level
   } deriving ( Eq, Show )
 
-class HasConfig a where
-  cId          :: Lens' a Config
-  cConfigFile  :: Lens' a (Maybe String)
-  cPort        :: Lens' a (Maybe Word32)
-  cTimeout     :: Lens' a (Maybe Word32)
-  cLogLevel    :: Lens' a (Maybe LogLevel)
+class HasConf a where
+  cId          :: Lens' a Conf
+  confFile     :: Lens' a (Maybe String)
+  confPort     :: Lens' a (Maybe Int)
+  confTimeout  :: Lens' a (Maybe Int)
+  confLogLevel :: Lens' a (Maybe LogLevel)
 
-  cConfigFile  = cId . lens _cConfigFile (\s a -> s { _cConfigFile = a } )
-  cPort        = cId . lens _cPort       (\s a -> s { _cPort = a } )
-  cTimeout     = cId . lens _cTimeout    (\s a -> s { _cTimeout = a } )
-  cLogLevel    = cId . lens _cLogLevel   (\s a -> s { _cLogLevel = a } )
+  confFile      = cId . lens _confFile     (\s a -> s { _confFile = a } )
+  confPort      = cId . lens _confPort     (\s a -> s { _confPort = a } )
+  confTimeout   = cId . lens _confTimeout  (\s a -> s { _confTimeout = a } )
+  confLogLevel  = cId . lens _confLogLevel (\s a -> s { _confLogLevel = a } )
 
-instance HasConfig Config where
+instance HasConf Conf where
   cId = id
 
-instance Default Config where
-  def = Config
-    { _cConfigFile = Just "/conf/dev.yaml"
-    , _cPort       = Just 3030
-    , _cTimeout    = Just 120
-    , _cLogLevel   = Just LevelInfo
+instance Default Conf where
+  def = Conf
+    { _confFile     = Just "/conf/dev.yaml"
+    , _confPort     = Just 3030
```

Comment:
Maybe make this `5000` vs `3030`? The `3030` and `4040` were totally pulled out of a hat - port numbers like `3000` and `5000` seem like common defaults.